src/TeamleaderDotNet/Utils/UnixTimestampExtensions.cs

```diff
@@ -44,7 +44,7 @@ public static DateTime UnixTimeToDateTime(this int unixtime)
         /// <returns>Returns a DateTime object that represents value of the Unix time.</returns>
         public static DateTime UnixTimeToDateTime(this string unixtime)
         {
-            return UnixTimeToDateTime(long.Parse(unixtime.ToString()));
+            return !string.IsNullOrEmpty(unixtime) ? UnixTimeToDateTime(long.Parse(unixtime)) : DateTime.MinValue;

```

Comment:
yes you are correct. I was already testing my implementation for custom fields and there i needed to convert unixtime to datetime but the value of the custom field was null so i had an error.. i'll update the pull request in a moment. thanks for the fast review and great work. It helped me already a lot. As soon as i have some time left, i'll do some other pull request in future.